### DISCUSSION TITLE
Enable LDAP Authentication

### DISCUSSION THREAD

**Discussion by randombdk** [ngoduykhanh/PowerDNS-Admin#1289] [Q&A]
Hi,

So never the less what we do - we can not keep Enable ldap authentication enabled - if i press save and page reload its removed.

Is this a bug with the button ? we tried both with ldap and ldaps no luck nor any logs.

We are using the latest docker build - the debug log dosent show any error during login only "LOCAL"

Do any documentation or example exists any where for this?

**Comment by randombdk** [comment]
This we Managed to solve ny enabling ldap through the SQL DB

Then it was working

**Comment by arbaldin** [comment]
I've done the same - enabled it in the mariadb directly. It seems something related to the frontend, but I can't confirm it.

**Comment by jbe-dw** [comment]
Hello,

There's a bug in the frontend Javascript introduced by changes in the Active Directory Page. I asked the author to repair it but he did not.
